specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -1045,3 +1045,10 @@ The goal of the Jakarta EE 9.1 release is to deliver a set of specifications fun
 Jakarta EE 9.1 is an extension to the foundational Jakarta EE 9 release.
 No API updates are expected in Jakarta EE 9.1.
 Only the Platform and Web Profile Specifications along with the TCKs and Compatible Implementations should be affected by Jakarta EE 9.1.
+
+=== Changes in Jakarta EE 10
+The goal of the Jakarta EE 10 release is to deliver a set of specifications that  and adding the support for the Java SE 11 and newer runtimes. The TCKs require support for both Java SE 11 and Java SE 17.
+
+Jakarta EE 10 is the first release in the Jakarta EE series to include major and minor component specification updates no limited to the javax to jakarta packge namespace change.
```

Comment:
Replace `updates no limited` with `updates not limited` above.  Also replace `packge` with `package`.

Reply:
Fixed.